src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -391,6 +408,21 @@
      */
     List<DimensionType> getDimensionTypes();
 
+    /**
+     * Gets the {@link Rotation} with the provided degrees.
+     *
+     * @param degrees The degrees of the rotation
+     * @return The {@link Rotation} with the given degrees or Optional.absent() if not found
+     */
+    Optional<Rotation> getRotationFromDegree(int degrees);
```

Comment:
The [`Rotation`](https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/util/rotation/Rotation.java) interface only provides the angle in degrees. There is a TODO marker to create an `Angle` interface which would be the appropriate place for adding radians, so that should be done as it's own PR/commit